index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="css.css">
+</head>
+<body>
+    <header id="header">
+        <img src="https://www.google.com/search?q=img&rlz=1C1BNSD_enAF999AF999&sxsrf=ALiCzsaINpAmuhP6Gb52DKWEamt7lZTMLQ:1653821668970&tbm=isch&source=iu&ictx=1&vet=1&fir=BD1Mn-7CrhkJeM%252CTuPHa6zZOTIdYM%252C_%253B2ZoxtX60fNTrTM%252CbqjMRiCbymtJHM%252C_%253BKhkAd_gxxRoWRM%252Cr6Vx2lLW0GRdzM%252C_%253B5ZLpMjDl9bjI_M%252ChuUCOMTfkRHQEM%252C_%253BOqPPyR4ESsagqM%252CAOn0wTjCfJqxcM%252C_%253BsQxJ7qukoOvyrM%252ChFtYl-FEX25j5M%252C_%253BRYBz6TYw2D7ZZM%252C2QpCn8mhLjh9DM%252C_%253BQqTdel_40i8TCM%252C7yXyQv4yffefoM%252C_%253BqE6ub8Rf_P0_gM%252Cwf7WyyD65HrStM%252C_%253BOmEuLe5xgrt5hM%252CrGnPI0Z2YGlEVM%252C_%253BL1nqsH2Tn58m5M%252CRPy7iEavyylz8M%252C_%253BGViRHWO97gwE9M%252CyMpQqf7dT0Eo_M%252C_%253BYJUC1kya5H7LpM%252CyMpQqf7dT0Eo_M%252C_%253B2n0NSjcDZsaH8M%252CyiJ5f9W4ifvuHM%252C_&usg=AI4_-kRHajVKOn34MLJxJuDRtb_9oMuvSw&sa=X&ved=2ahUKEwiz8OboxYT4AhVkSvEDHSy3AloQ9QF6BAgDEAE&biw=958&bih=951&dpr=1#imgrc=BD1Mn-7CrhkJeM" alt="" id="header-img">
+        <nav id="nav-bar">
+            <a class="nav-link" href="#form"></a>
+            <a class="nav-link" href="#form"></a>
+            <a class="nav-link" href="#form"></a>
+        </nav>
+    </header>
+    <video src="" id="video"></video>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+        <input type="email" name="email" id="email" placeholder="">
+        <input class="submit" type="submit" name="" id="submit">
+        
```

Comment:
Kindly follow the Stylelint technic for solving this kind of spacing. I think the spacing is not got like this.